Projects/UOContent/Items/Weapons/Axes/BaseAxe.cs

```diff
@@ -71,7 +52,7 @@ public override void UnscaleDurability()
 
             var scale = GetUsesScalar();
 
-            m_UsesRemaining = (m_UsesRemaining * 100 + (scale - 1)) / scale;
+            _usesRemaining = (_usesRemaining * 100 + (scale - 1)) / scale;
```

Comment:
Assignments must be done with the property so the object gets marked dirty.

Reply:
:+1: And do we need the `InvalidateProperties();` call underneath or is that handled by the InvalidateProperties attribute?

Reply:
It's handled by the attribute too.